test/unit/io/structure_file/format_vienna_test.cpp

```diff
@@ -55,16 +58,16 @@ struct read : public ::testing::Test
         { "example 2" }
     };
 
-    std::vector<rna4_vector> const expected_seq
+    std::vector<rna5_vector> const expected_seq
     {
-        { "GCGGAUUUAGCUCAGUUGGGAGAGCGCCAGACUGAAGAUUUGGAGGUCCUGUGUUCGAUCCACAGAAUUCGCA"_rna4 },
-        { "UUGGAGUACACAACCUGUACACUCUUUC"_rna4 }
+        { "GCGGAUUUAGCUCAGUUGGGAGAGCGCCAGACUGAAGAUUUGGAGGUCCUGUGUUCGAUCCACAGAAUUCGCA"_rna5 },
+        { "UUGGAGUACACAACCUGUACACUCUUUC"_rna5 }
     };
 
-    std::vector<std::vector<dot_bracket3>> const expected_structure
+    std::vector<std::vector<wuss<51>>> const expected_structure
     {
-        { "(((((((..((((........)))).((((.........)))).....(((((.......))))))))))))."_db3 },
-        { "..(((((..(((...)))..)))))..."_db3 }
+        { "(((((((..((((........)))).((((.........)))).....(((((.......))))))))))))."_wuss51 },
+        { "..(((((..(((...)))..)))))..."_wuss51 }
```

Comment:
Why not rna4 and db3?

Reply:
Simply because the others are the default and I would need to add an extra traits object to the test and then always specify all template arguments of the sequence_file_input object. I think the test is just as fine now?
We should think about adding tests for different alphabets though.

Reply:
Sure, it is fine like this! I was just curious why it was changed, as the structure uses only a tiny subset of the wuss alphabet now. But I see the benefit of omitting the template arguments :)